crates/cheatcodes/src/inspector.rs

```diff
@@ -1419,7 +1422,23 @@ fn process_create<DB: DatabaseExt>(
             }
 
             call.caller = DEFAULT_CREATE2_DEPLOYER;
+        }
+    }
+    Ok(())
+}
 
+fn process_create<DB: DatabaseExt>(
```

Comment:
Could we add some docs on what the two functions do now? as we have `apply_create2_deployer` and `process_create`, and previously `process_create` was fallible, but now it will always succeed and update state in case of scripts.

Reply:
@Evalir Just bumping since this is ready for review. Thanks.